config.example.yaml

```diff
@@ -14,6 +14,11 @@ oper:
   # oper via CHALLENGE key instead
   #file: ~/libera.key
 
+bancmd: "KLINE 1440 {ident}@{user.ip} :{reason}"
```

Comment:
I don't think `{user.ip}` will work here - I think `.format` vars have to be simple key/value

Reply:
It seemed to work alright when I tested it

Reply:
perhaps i tested poorly. will try again!

Reply:
well I stand corrected.

```
>>> class A:
...     a=1
...
>>> a = A()
>>> "{a.a}".format(a=a)
'1'
```